rasa/constants.py

```diff
@@ -12,6 +12,8 @@
 DEFAULT_NLU_RESULTS_PATH = "nlu_comparison_results"
 DEFAULT_REQUEST_TIMEOUT = 60 * 5  # 5 minutes
 
+NLU_FORMAT_UNKNOWN = "unk"
```

Comment:
I'd rather reuse the constant from `rasa/nlutraining_data/loading.py` instead of defining a new one.

Reply:
ah ok, I'd rather keep it in the old module. if you don't want to import the constant from the other module you could also write a short function `is_nlu_file` to `loading`.

Reply:
Done, the solution was to move back `is_nlu_file` to `data.py` and make it  `_is_nlu_flie` import `guess_format` from `loading.py`